src/BenchmarksApps/SslStream/Server/Program.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Net;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Net.Sockets;
+using System.CommandLine;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Crank.EventSources;
+
+using SslStreamServer;
+using SslStreamCommon;
+
+internal class Program
+{
+    private static int s_errors;
+
+    private static async Task<int> Main(string[] args)
+    {
+        var rootCommand = new RootCommand("SslStream benchmark server");
+        OptionsBinder.AddOptions(rootCommand);
+        rootCommand.SetHandler<ServerOptions>(Run, new OptionsBinder());
+        return await rootCommand.InvokeAsync(args).ConfigureAwait(false);
+    }
+
+    static async Task Run(ServerOptions options)
+    {
+        SetupMeasurements();
+
+        var sslOptions = CreateSslServerAuthenticationOptions(options);
+        using var sock = new Socket(SocketType.Stream, ProtocolType.Tcp);
+        sock.Bind(new IPEndPoint(IPAddress.IPv6Any, options.Port));
+        sock.Listen();
+
+        Log($"Listening on {sock.LocalEndPoint}.");
+
+        CancellationTokenSource ctrlC = new CancellationTokenSource();
+        Console.CancelKeyPress += (s, e) =>
+        {
+            Log("Shutting down...");
+            e.Cancel = true;
+            ctrlC.Cancel();
+        };
+
+        try
+        {
+            while (!ctrlC.IsCancellationRequested)
+            {
+                var clientSock = await sock.AcceptAsync(ctrlC.Token).ConfigureAwait(false);
+                _ = Task.Run(() => ProcessClient(clientSock, options, sslOptions, ctrlC.Token));
+            }
+        }
+        catch (OperationCanceledException)
+        {
+            // Expected, just return
+        }
+
+        LogMetric("sslstream/errors", s_errors);
```

Comment:
Does this _actually_ report the error count when there are any? 🤔 I mean, within the Crank environment

Reply:
Yes